### ISSUE TITLE
Outdated Java-Packages

### ISSUE THREAD

**Issue by Mechtilde** [openjverein/jverein#19]
Hello,

most of the Java-packages shipped in /lib are outdated.

I package it for Debian and did it with more recent packages.

I can document here which version I use?

Kind regards

Mechtilde

**Comment by kpatzwald** [comment]
Yes! That would be very kind. :-)

**Comment by almereyda** [comment]
My suggestion would be to rescope this issue towards the suggestion by @tobidope in #9 and to escalate this into a *lifecycle management* story, including a first iteration of said procedure.

**Comment by dippeal** [comment]
Neue libs siehe https://github.com/openjverein/jverein/pull/57

**Comment by tolot27** [comment]
Kan dieser Issue dann geschlossen werden, da ja #57 merged ist.

**Comment by Mechtilde** [comment]
Das Problem ist damit erstmal gelöst. Sollte es erneut Pakete geben, die sich der Version unterscheiden, werde ich neue Issues erstellen.

**Comment by tolot27** [comment]
> > Kan dieser Issue dann geschlossen werden, da ja #57 merged ist.
> 
> Der PR ist closed (nicht merged), da mir keiner die reviews gegeben hat. Ich schaue mir die libs noch mal an und erstelle ggf. einen PR.

Richtig, #57 ist nicht merged, allerdings closed, da e91dcc8 merged ist. Daher hatte ich transitiv angenommen, dass dieser Issue damit gelöst ist. 😏 

PS: In anderen GitHub-Projekten verlinke ich die Issues und PRs auf der rechten Seite (desktop view), damit die Verknüpfung klar ist.
